crypto/evp/m_sigver.c

```diff
@@ -480,14 +480,14 @@ int EVP_DigestSignFinal(EVP_MD_CTX *ctx, unsigned char *sigret,
     if (sigret == NULL || (ctx->flags & EVP_MD_CTX_FLAG_FINALISE) != 0)
         return pctx->op.sig.signature->digest_sign_final(pctx->op.sig.algctx,
                                                          sigret, siglen,
-                                                         (siglen == NULL) ? 0 : *siglen);
+                                                         sigret == NULL ? 0 : *siglen);
```

Comment:
Should we actually check *both* `sigret` and `siglen`? i.e. `(sigret == NULL || siglen == NULL) ? 0 : *siglen`

Reply:
The siglen must always be non-NULL in the calls. We do not do such checks in other places.

Reply:
For example there is no check for non-NULL ctx either in the function.

Reply:
ok